### DISCUSSION TITLE
Table records per page

### DISCUSSION THREAD

**Discussion by kendal-kirby** [twilio-labs/paste#1846] [Q&A] [answered]
Is there currently any specific guidance on how many records to display in a table per page?

**Comment by nkrantz** [comment, marked as answer]
Hi @kendal-kirby,

Check out [this section of our Pagination docs](https://paste.twilio.design/components/pagination#determining-how-many-items-to-show-per-page). It includes some general guidance and also ideas on how to scale up/down number of records per page. Doing some user testing will be the best way to get a good answer for your specific use case!
